src/read/pdb/parser.rs

```diff
@@ -18,9 +19,17 @@ use std::io::{BufRead, BufReader};
 /// # Related
 /// If you want to open a file from memory see [`open_pdb_raw`]. There is also a function to open a mmCIF file directly
 /// see [`crate::open_mmcif`]. If you want to open a general file with no knowledge about the file type see [`crate::open`].
-pub fn open_pdb(
+pub fn open_pdb(filename: impl AsRef<str>) -> Result<(PDB, Vec<PDBError>), Vec<PDBError>> {
+    open_pdb_with_options(filename, &ReadOptions::default())
+}
+
+/// Parse the given file into a PDB struct with [`ReadOptions`].
+///
+/// # Related
+/// See [`open_pdb`] for a version of this function with sane defaults.
+pub fn open_pdb_with_options(
```

Comment:
Should this be made `pub(crate)`? Since we want people to use a single API, through methods on the `ReadOptions` struct

Reply:
Marking these functions as deprecated in this release and making them `pub(crate)` might be easier for people to make the transition? Making them `pub(crate)` now would break all existing code.

Reply:
Ok cool. Mark as deprecated